t/plugin/proxy-rewrite.t

```diff
@@ -34,7 +34,61 @@ run_tests;
 
 __DATA__
 
-=== TEST 1: sanity
+=== TEST 1: set route(rewrite host)
```

Comment:
This case is samiliar to the one in proxy-rewrite-3.t?

Reply:
Done